src/pkg/packaging/windows/package.targets

```diff
@@ -22,11 +22,10 @@
             Condition="'$(OSGroup)' == 'Windows_NT' and '$(GenerateMSI)' == 'true'">
       
       <PropertyGroup>
-        <WixVersion Condition="'$(WixVersion)' == ''">3.10.2</WixVersion>
         <WixFilename>wix.$(WixVersion).zip</WixFilename>
       </PropertyGroup>
 
-      <Exec Command="powershell -NoProfile -NoLogo $(GetWixScript) -WixVersion $(WixVersion) -OutputDir $(WixToolsDir)" />
+      <Exec Command="powershell -NoProfile -NoLogo $(GetWixScript) -WixVersion $(WixVersion) -WixFilename $(WixFilename) -OutputDir $(WixToolsDir)" />
```

Comment:
Sorry, I didn't look at this closer last time I reviewed it.

`-WixVersion` isn't a parameter to the `GetWix.ps1` script.  Can you remove that parameter?  I thought that the existing parameter was being passed in for the `[string]$WixFilename="wix.3.10.4.zip"` parameter.

So this line can be:

```
<Exec Command="powershell -NoProfile -NoLogo $(GetWixScript) -WixFilename $(WixFilename) -OutputDir $(WixToolsDir)" />
```